variables.tf

```diff
@@ -27,6 +27,10 @@ variable "lb_ingress_cidr" {
   description = "CIDR to allow access to this load balancer. Allows white listing of IPs if you need that kind of thing, otherwise it just defaults to erebody."
 }
 
+variable "lb_security_group_id" {
+  description = "The id of the ECS cluster load balancer security group."
+}
+
```

Comment:
Find a better name for this guy. If it's the ECS cluster LB SG then maybe `cluster_sg_id`?